config_defaults.yml

```diff
@@ -556,46 +556,43 @@ Rakefile:
 Gemfile:
   required:
     ':development':
-      # hardcode JSON version to what's shipped in pdk for now
-      - gem: json
-        version: '= 2.0.4'
-        condition: "Gem::Requirement.create('~> 2.4.2').satisfied_by?(Gem::Version.new(RUBY_VERSION.dup))"
-      - gem: json
-        version: '= 2.1.0'
-        condition: "Gem::Requirement.create(['>= 2.5.0', '< 2.7.0']).satisfied_by?(Gem::Version.new(RUBY_VERSION.dup))"
-      - gem: json
-        version: '= 2.3.0'
-        condition: "Gem::Requirement.create(['>= 2.7.0', '< 2.8.0']).satisfied_by?(Gem::Version.new(RUBY_VERSION.dup))"
-      - gem: 'puppet-module-posix-default-r#{minor_version}'
-        version: '~> 1.0'
+      - gem: 'json'
+      - gem: 'voxpupuli-puppet-lint-plugins'
+        version: '>= 3.0.0'
+      - gem: 'facterdb'
+        version: '< 2.0.0'
+      - gem: 'metadata-json-lint'
+        version: '< 4.0.0'
+      - gem: 'puppet-lint'
+        version: '< 3.0.0'
+      - gem: 'puppetlabs_spec_helper'
+        version: '< 4.0.0'
+      - gem: 'rspec-puppet'
+        version: '< 3.0.0'
```

Comment:
`rspec-puppet` is also redundant IMHO: puppetlabs_spec_helper depends on it (https://github.com/puppetlabs/puppetlabs_spec_helper/blob/a28e6e9c761d186399577dcb86e80cfae3c302b6/puppetlabs_spec_helper.gemspec#L29).

Reply:
removed